### PR TITLE
feat(gatsby-source-wordpress): Ability to set a custom rest api route prefix

### PR DESCRIPTION
**Pull request by crock**
## Description

Adds a configuration option called `restApiRoutePrefix` that allows you to set a custom route prefix if one of your WordPress plugins changed it from `wp-json`. `wp-json` is left as the default value.

## Related Issues

https://github.com/gatsbyjs/gatsby/issues/20995

### CODE REVIEW

File: packages/gatsby-source-wordpress/src/gatsby-node.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ exports.sourceNodes = async (
   _includedRoutes = includedRoutes
   _excludedRoutes = excludedRoutes
   _keepMediaSizes = keepMediaSizes
+  _restApiRoutePrefix = restApiRoutePrefix
```
Comment (TylerBarnes) [inline packages/gatsby-source-wordpress/src/gatsby-node.js]: `_restApiRoutePrefix` is undefined, it needs to be added above where the other _vars are declared.

Comment (crock) [reply packages/gatsby-source-wordpress/src/gatsby-node.js]: Done! :)

Comment (TylerBarnes) [reply packages/gatsby-source-wordpress/src/gatsby-node.js]: Awesome, thanks!

**Comment by TylerBarnes** [comment]
Ahh, couldn't figure out why I couldn't merge this. @gatsbyjs/learning needs to review since the README was updated, then we should be good. Thanks again @crock 👍

**Pull request merged**